llama_stack/providers/remote/inference/vllm/vllm.py

```diff
@@ -180,6 +181,16 @@ async def _get_params(
                 self.formatter,
             )
 
+        if fmt := request.response_format:
+            if fmt.type == ResponseFormatType.json_schema.value:
+                input_dict["extra_body"] = {
+                    "guided_json": request.response_format.json_schema
+                }
```

Comment:
Note that using

```python
input_dict["response_format"] = {
  "type": "json_schema",
  "json_schema": {
    "name": "example_name",
    "schema": request.response_format.json_schema
  }
}
```

as per the usual OpenAI api, doesn't work.

Reply:
Can you document it in an vLLM issue and link it back here?